index/scorch/scorch.go

```diff
@@ -263,7 +263,7 @@ func (s *Scorch) Close() (err error) {
 		err = s.rootBolt.Close()
```

Comment:
This err gets overwritten. Is that ok? Should we just return here if this err isn't nil perhaps?

Reply:
i didn't see the above initialisation.. oops..